README.md

```diff
@@ -84,8 +51,3 @@ Contributions, issues and feature requests are welcome!<br />Feel free to check
 
 Copyright © 2020-Present [St. Jude Cloud Team](https://github.com/stjudecloud).<br />
 This project is [MIT](https://github.com/stjudecloud/workflows/blob/master/LICENSE.md) licensed.
-
-[wdl]: http://openwdl.org/
-[cromwell]: https://github.com/broadinstitute/cromwell
-[bioconda]: https://bioconda.github.io/
-[oliver]: https://github.com/stjudecloud/oliver
```

Comment:
I actually like having this, but I'd make it a section. Link to the WDL spec, `miniwdl`, `pytest-workflows`, and anything else we think is an important part of the WDL ecosystem.

Reply:
To be clear, I'd remove these links and do a replacement with more relevant ones.

Reply:
Alright. No disagreement. Will add that first thing in the AM.